dev-docs/bidders/adikteev.md

```diff
@@ -2,18 +2,21 @@
 layout: bidder
 title: Adikteev
 description: Prebid Adikteev Bidder Adaptor
+
 top_nav_section: dev_docs
 nav_section: reference
+
 hide: true
+
 biddercode: adikteev
-prebid_1_0_supported : true
```

Comment:
why delete this variable? It currently needs to be there or the adapter will not be downloadable.

Reply:
Done.